.classpath

```diff
@@ -7,12 +7,15 @@
 		</attributes>
 	</classpathentry>
 	<classpathentry kind="lib" path="build"/>
-	<classpathentry kind="lib" path="lib/unohelper-4.0.3-SNAPSHOT-jar-with-dependencies.jar"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11">
 		<attributes>
 			<attribute name="module" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="con" path="org.libreoffice.ide.eclipse.java.OOO_CONTAINER/LibreOffice 7.2"/>
+	<classpathentry kind="lib" path="lib/commons-lang3-3.11.jar"/>
+	<classpathentry kind="lib" path="lib/guava-29.0-jre.jar"/>
+	<classpathentry kind="lib" path="lib/jsoup-1.15.3.jar"/>
```

Comment:
These don't seem to exist in this repo?

Reply:
I had added these JARs under lib to my fork.

Reply:
Ok can you remove them from this PR then please?

Reply:
I think we misunderstood each other.  I copied the three jar libraries into the lib directory and included them in the lib path. They must be included, otherwise the update to the latest version of unohelper will not work.

Reply:
Ah I see, looks good then.